docs/reference/deploy/master-config-reference.rst

```diff
@@ -103,6 +103,12 @@ specified here. You can customize environment variables for CUDA (NVIDIA GPU), C
 GPU) tasks differently by specifying a dict with ``cuda`` (``gpu`` prior to Determined 0.17.6),
 ``cpu``, and ``rocm`` keys.
 
+``startup_hook``
+================
+
+An optional inline script that will be executed as part of task set up. This is defined under
```

Comment:
I know you had a seperate pr but helm / helm docs?

If you wanna do seperate that is fine but I think I prefer one PR

Reply:
that'll take me longer to test potentially not until tomorrow. I think it'd make sense to make sure the main changes are tested and landed asap?